dagql/idtui/frontend_pretty.go

```diff
@@ -623,6 +632,8 @@ func (fe *frontendPretty) keys(out *termenv.Output) []key.Binding {
 		key.NewBinding(key.WithKeys("r"),
 			key.WithHelp("r", "go to error"),
 			KeyEnabled(focused != nil && focused.ErrorOrigin != nil)),
+		key.NewBinding(key.WithKeys("t"),
```

Comment:
This should probably be conditional on whether the focused span is terminal-able, like above